test/helpers.bash

```diff
@@ -261,5 +279,5 @@ EOF
   stacker clean
   rm -rf folder1
   cd "$ROOT_DIR"
-  rm -rf "tmpdir"
+  rm -rf "$tmpdir"
```

Comment:
worth mentioning you fixed a case where we left a tmpdir around when using `test_copy_buffer_size` I think.